### PR TITLE
Fix stale docs about LB annotation

### PR DESCRIPTION
**Pull request by thisisharrsh**
…ernal-load-balancers

Fixes : #46575

### CODE REVIEW

**Comment by thisisharrsh** [comment]
_Deployment preview : https://deploy-preview-46590--kubernetes-io-main-staging.netlify.app/docs/reference/labels-annotations-taints/#node-kubernetes-io-exclude-from-external-load-balancers_

**Review by sftim — COMMENTED** [review]
/retitle Fix stale docs about LB annotation
/lgtm

File: content/en/docs/reference/labels-annotations-taints/_index.md [hunk 1]
```diff
@@ -627,9 +627,7 @@ Example: `node.kubernetes.io/exclude-from-external-load-balancers`
 
 Used on: Node
 
-Kubernetes automatically enables the `ServiceNodeExclusion` feature gate on
-the clusters it creates. With this feature gate enabled on a cluster,
-you can add labels to particular worker nodes to exclude them from the list of backend servers.
+You can add labels to particular worker nodes to exclude them from the list of backend servers used by external load balancers.
 The following command can be used to exclude a worker node from the list of backend servers in a
 backend set:
```
Comment (sftim) [inline content/en/docs/reference/labels-annotations-taints/_index.md:632]: Does this command _always_ work, or is it only applicable with certain cloud provider integrations? Let's try to document that.

**Review by sftim — COMMENTED** [review]
Even with a caveat, this is an improvement.

/lgtm
/approve

File: content/en/docs/reference/labels-annotations-taints/_index.md (hunk 1 above)
Comment (thisisharrsh) [reply content/en/docs/reference/labels-annotations-taints/_index.md:632]: Yes, we should try to test this command with different cloud environment. And later we can update the docs

**Pull request merged**